python_modules/dagster/dagster/_core/definitions/declarative_scheduling/asset_condition.py

```diff
@@ -393,6 +393,24 @@ def parents_updated_since_cron(cron_schedule: str, timezone: str = "UTC") -> "As
             )
         )
 
+    @staticmethod
+    def any_deps_match(condition: "AssetCondition") -> "AssetCondition":
+        """Returns an AssetCondition that is true for an asset partition if at least one partition
+        of any of its dependencies evaluate to True for the given condition.
+        """
+        from .dep_condition import AnyDepsCondition
+
+        return AnyDepsCondition(operand=condition)
+
+    @staticmethod
+    def all_deps_match(condition: "AssetCondition") -> "AssetCondition":
+        """Returns an AssetCondition that is true for an asset partition if at least one partition
+        of all of its dependencies evaluate to True for the given condition.
+        """
+        from .dep_condition import AllDepsCondition
+
+        return AllDepsCondition(operand=condition)
+
```

Comment:
I assume we will change this to `SchedulingCondition`?

Reply:
That's right yep